.github/workflows/aws_openshift_rosa_hcp_single_region_tests.yml

```diff
@@ -0,0 +1,646 @@
+                        for file in registry.yml identity.yml; do
+                          yq ". *d load(\"generic/kubernetes/single-region/tests/helm-values/$file\")" values.yml > values-result.yml
+                          cat values-result.yml && mv values-result.yml values.yml
+                        done
+                      fi
+
+                      ./generic/openshift/single-region/procedure/assemble-envsubst-values.sh
+
+
+            - name: 🏁 Install Camunda 8 using the generic/openshift helm chart procedure
+              run: |
+                  set -euxo pipefail # tolerate, nothing.
+
+                  source generic/openshift/single-region/procedure/chart-env.sh
+                  source generic/openshift/single-region/procedure/generate-passwords.sh
+
+                  ./generic/openshift/single-region/procedure/create-identity-secret.sh
+
+                  # Generate tests objects
+                  if [ "$TESTS_ENABLED" == "true" ]; then
+                    # Create the pull secrets described in generic/kubernetes/single-region/tests/helm-values/registry.yml
+                    kubectl create secret docker-registry index-docker-io \
+                        --docker-server=index.docker.io \
+                        --docker-username="${{ steps.secrets.outputs.DOCKERHUB_USER }}" \
+                        --docker-password="${{ steps.secrets.outputs.DOCKERHUB_PASSWORD }}" \
+                        --namespace="$TEST_NAMESPACE"
+
+                    kubectl create secret generic identity-secret-for-components-integration \
+                        --from-literal=identity-admin-client-id="${{ steps.secrets.outputs.CI_CAMUNDA_USER_TEST_CLIENT_ID }}" \
+                        --from-literal=identity-admin-client-secret="${{ steps.secrets.outputs.CI_CAMUNDA_USER_TEST_CLIENT_SECRET }}" \
+                        --namespace="$TEST_NAMESPACE"
+                  fi
+
+                  ./generic/openshift/single-region/procedure/install-chart.sh
+
+            - name: 👀⏳ Wait for the deployment to be healthy using generic/kubernetes/single-region
+              timeout-minutes: 10
+              run: |
+                  ./generic/kubernetes/single-region/procedure/check-deployment-ready.sh
+
+            - name: 🧙‍♂️ Clone camunda/camunda-platform-helm
+              if: env.TESTS_ENABLED == 'true'
+              uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+              with:
+                  repository: camunda/camunda-platform-helm
+                  ref: ${{ env.TESTS_CAMUNDA_HELM_CHART_REPO_REF }}
+                  path: ${{ env.TESTS_CAMUNDA_HELM_CHART_REPO_PATH }}
+                  fetch-depth: 0
+
+            - name: 🧪 TESTS - Set variables
+              if: env.TESTS_ENABLED == 'true'
+              run: |
+                  set -euxo pipefail # tolerate, nothing.
+
+                  CAMUNDA_VERSION=$(cat .camunda-version)
```

Comment:
interesting, now I get the camunda-version thing.
I was thinking we just use the upstream chart that is released.